indexer/httpd/src/graphql/types/tendermint.rs

```diff
@@ -23,6 +27,19 @@ impl From<tx_sync::Response> for TxSyncResponse {
             data: STANDARD.encode(resp.data),
             log: resp.log,
             hash: resp.hash.to_string(),
+            codespace: resp.codespace,
+        }
+    }
+}
+
+impl From<TxSyncResponse> for tx_sync::Response {
+    fn from(resp: TxSyncResponse) -> Self {
+        tx_sync::Response {
+            code: tendermint::abci::Code::from(resp.code),
+            codespace: resp.codespace,
+            data: Binary::from_str(&resp.data).unwrap().into_inner().into(),
+            log: resp.log,
+            hash: tendermint::hash::Hash::from_str(&resp.hash).unwrap(),
```

Comment:
There seems to be no use for this code. I deleted it and everything still works.